common/Android.mk

```diff
@@ -94,6 +94,10 @@ LOCAL_CPPFLAGS += \
         -D_GNU_SOURCE=1 -D_FILE_OFFSET_BITS=64 \
         -O3
 
+ifneq ($(strip $(BOARD_DISABLE_DRM_CTM)), true)
+LOCAL_CPPFLAGS += -DUSE_DRM_CTM
```

Comment:
@Kishore409 Take note, We need to enable this flag BOARD_DISABLE_DRM_CTM in some of our Board Configs for APL. We don't need this flag for KBL or other boards.

Reply:
@Kishore409 ping

Reply:
Sure @kalyankondapally